G/Geant4/build_tarballs.jl

```diff
@@ -0,0 +1,83 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "Geant4Builder"
+version = v"0.1.0"
+
+# Collection of sources required to build Geant4Builder
+sources = [
+    "https://github.com/Geant4/geant4/archive/v10.5.1.tar.gz" =>
+    "443efb0d16e8a5fd195176573d21d2e12415ae7853dd39cc0517171aea243227",
+
+    "https://github.com/libexpat/libexpat/releases/download/R_2_2_8/expat-2.2.8.tar.gz" =>
+    "bd507cba42716ca9afe46dd3687fb0d46c09347517beb9770f53a435d2c67ea0",
+
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd expat-2.2.8/
+./configure --prefix=$prefix --host=$target
+make
+make install
+cd ../geant4-10.5.1/
+mkdir build
+cd build/
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=/opt/$target/$target.toolchain ..
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:x86_64, libc=:glibc),
+    MacOS(:x86_64),
+    FreeBSD(:x86_64)
+]
```

Comment:
I take it this doesn't build on Windows?  What about i686 Linux, or musl?

Reply:
I think it should work with 32-bit platforms, on the website the pre-built library is actually for 32-bit windows: https://geant4.web.cern.ch/support/download